packages/components/src/theme/ThemeUtils.test.ts

```diff
@@ -333,6 +336,47 @@ describe('getDefaultBaseThemes', () => {
       },
     ]);
   });
+
+  /**
+   * This test is to ensure that the css variables in theme-dark and theme-light are in sync,
+   * to prevent the case where a css variable is added to one theme and not the other.
+   */
+  it('light and dark themes should contain the same css variables', () => {
+    const themeDarkDir = './packages/components/src/theme/theme-dark';
+    const themeLightDir = './packages/components/src/theme/theme-light';
+
+    function extractCssVariablesFromFiles(directory: string): Set<string>[] {
+      const files = fs.readdirSync(directory);
+      const fileVariables = [] as Set<string>[];
+      files.forEach(file => {
+        const variables = new Set<string>();
+        const filePath = path.join(directory, file);
+        const fileContent = fs.readFileSync(filePath, 'utf8');
+        // strip comments as they may contain css variable names
+        const fileContentsNoComments = fileContent.replace(
+          /\/\*[\s\S]*?\*\//g,
+          ''
+        );
+        const lines = fileContentsNoComments.split('\n');
+        // extract css variables from each line, before the colon
+        const matches = lines.map(line => line.match(/(--[a-zA-Z-]+):/)?.[1]);
```

Comment:
It doesn't look like we use this eslint rule in our codebase, but in the past I was led to believe that `RegEx.prototype.exec` may be preferred over `String.prototype.match`. Up to you whether you care on this one.

https://typescript-eslint.io/rules/prefer-regexp-exec/